app/models/patient.rb

```diff
@@ -55,6 +56,7 @@ class Patient
   field :other_phone, type: String
   field :other_contact_relationship, type: String
   field :identifier, type: String
+  field :random_pin, type: Integer
```

Comment:
Can you help me understand why the new field rather than pressing identifier into service? Are we going to maintain both fields?

Reply:
Yes - the plan right now is to maintain both fields for several months to give clinics + CMs a chance to adjust and to avoid disruption. If we replaced it, then we'd risk issues with existing pledges